Client/Frontend/Settings/SettingsRowViews.swift

```diff
@@ -39,6 +39,17 @@ extension Row {
             uuid: option.key
         )
     }
+    
+    /// Creates a switch toggle `Row` which updates a `Preferences.Option<Bool>`
+    static func boolRow(title: String, detailText: String, option: Preferences.Option<Bool>, onValueChange: SwitchAccessoryView.ValueChange? = nil) -> Row {
+        return Row(
+            text: title,
+            detailText: detailText,
+            accessory: .view(SwitchAccessoryView(initialValue: option.value, valueChange: onValueChange ?? { option.value = $0 })),
+            cellClass: MultilineSubtitleCell.self,
+            uuid: option.key
+        )
+    }
```

Comment:
Was confused that I thought this was a duplicate function. Probably best to do 1 of the 2 solutions:

1. Remove this function, add a `detailText: String? = nil` arg to the original function, adjust `cellClass` based on the `detailText` being nil
2. Rename this to `subtitleBoolRow` or `boolRowWithSubtitle` or something

Reply:
Fixed.